src/components/ListV2/ListRow.tsx

```diff
@@ -0,0 +1,216 @@
+import styled from '@emotion/styled'
+import {
+  ChangeEventHandler,
+  MouseEventHandler,
+  ReactNode,
+  useEffect,
+} from 'react'
+import type { Color } from '../../theme'
+import Checkbox from '../Checkbox'
+import Icon from '../Icon'
+import { ListCell } from './ListCell'
+import { useListContext } from './ListContext'
+
+const StyledListCell = styled(ListCell)`
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  grid-col-start: -1;
+`
+
+const StyledRow = styled('div', {
+  shouldForwardProp: prop => prop !== 'template',
+})<{
+  template: string
+  'data-disabled': boolean
+  'data-hoverable': boolean
+  'data-highlight': boolean
+  'data-variant': Color | undefined
+}>`
+  position: relative;
+  display: grid;
+  grid-template-columns: ${({ template }) => template};
+  align-items: center;
+  border: 1px solid ${({ theme }) => theme.colors.neutral.borderWeak};
+  border-radius: ${({ theme }) => theme.radii.default};
+  transition: box-shadow 200ms ease, border-color 200ms ease;
+  background-color: ${({ theme }) => theme.colors.neutral.background};
+  cursor: auto;
+  font-size: 14px;
```

Comment:
Two possibilities:
1. You get from theme
2. You make a styled of `Text` as `div`